packages/driver-interface/services/bot.js

```diff
@@ -18,19 +18,16 @@ const onLogin = (vehicleId, ctx) => {
 
   ctx.metadata.setVehicleIdFromTelegramId(telegramId, vehicleId)
 
-  if (currentVehicle && currentVehicle.telegramId) {
+  if (currentVehicle.telegramId) {
     return
   }
 
-  if (currentVehicle) {
-    setInstructions(currentVehicle.id, currentVehicle.activities.slice(1, -1))
-    addVehicle(vehicleId, {
-      ...currentVehicle,
-      telegramId,
-    })
-  } else {
```

Comment:
reason why this was done was that you might should be able to register before you had a plan. But we don't have the rest of that functionality yet, so good choice to remove it.

Reply:
The currentVehicle check already existed on line 14, so I just cleaned it up :)